server/space-service/src/main/java/edu/illinois/ncsa/incore/service/space/controllers/UsageController.java

```diff
@@ -115,15 +133,10 @@ public String getUsage(@HeaderParam("x-auth-userinfo") String userInfo) {
     @Operation(summary = "Gives the usage status of the given username.",
         description = "This will only work for admin user group.")
```

Comment:
It's not related to what you trying to do in this PR but I'm wondering if we should follow the same pattern of allocations endpoint to change this path to ` @Path("users/{username}")`

See 
```
 @GET
    @Path("users/{username}")
    @Produces(MediaType.APPLICATION_JSON)
    @Operation(summary = "Gives the allocation status of the given username.",
    ```

Reply:
Or the other way around change allocations to just @Path("{username}")

Reply:
Good catch! I agree this makes the codes more consistent. I will change it to `@Path("users/{username}")` as AllocationsController needs to handle groupname as well.

Reply:
I think I also need to change it in the swagger document as well